src/styles/form.style.scss

```diff
@@ -1,30 +1,90 @@
 .container {
     width: 100%;
-    height: 500px;
+    height: 570px;
     display: flex;
     justify-content: center;
     align-items: center;
-    form {
-        width: 300px;
-        height: 200px;
-        border: 2px solid blue;
-        border-radius: 10px;
-        vertical-align: middle;
+    background-color: rgb(235, 227, 227);
+    font-family: Arial, Helvetica, sans-serif;
+    .box {
+        width: 350px;
+        height: 400px;
+        background-color: white;
+        box-shadow: -2px -2px rgb(219, 213, 213);
         display: flex;
-        flex-direction: column;
         justify-content: center;
         align-items: center;
-        gap: 5px;
-        input {
-            width: 150px;
-            outline-color: blue;
-        }
-        input[type = "submit"] {
-             margin-top: 20px;
-        }
-        p{
-            color: red;
-            font-size: 10px;
+        form {
+            width: 300px;
+            height: 330px;
+            .header {
+                display: flex;
+                justify-content: space-between;
+                margin-bottom: 22px;
+                span {
+                    font-size: 25px;
+                    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+                }
+                .icons {
+                    display: flex;
+                    justify-content: center;
+                    align-items: center;
+                    gap: 10px;
+                    svg {
+                        color: gray;
+                        border: 1px solid gray;
+                        border-radius: 50%;
+                        padding: 2px;
+                        &:hover {
+                            opacity: 0.5;
+                            cursor: pointer;
+                        }
+                    }
+                }
+            }
+            label {
```

Comment:
You need to have main.scss and then import other stylesheets in it. For instance you need to have a separate input.scss file